includes/alerts/transport.jira.php

```diff
@@ -58,7 +59,10 @@
 $ret  = curl_exec($curl);
 $code = curl_getinfo($curl, CURLINFO_HTTP_CODE);
 if( $code == 200 ) {
-	return true;
+    $jiraout = json_decode($ret, true);
+    log_event("Created jira issue ".$jiraout['key']." for ".$device, $device, 'poller', 5);
```

Comment:
I'm sorry, but I'll have to reopen this. There's bug in the transport file. And I'm too lazy to open another pr (as long as you're ok with it)